cirq/ops/__init__.py

```diff
@@ -26,10 +26,10 @@
     measure,
     measure_each,
     MeasurementGate,
-    Rot11Gate,
-    RotXGate,
-    RotYGate,
-    RotZGate,
+    CZPowGate,
```

Comment:
we've been using alphabetical and this breaks that style though I agree it's nice to have these near each other

Reply:
Done.